src/olympia/editors/tests/test_models.py

```diff
@@ -580,18 +577,19 @@ def test_award_points_bonus(self):
     def test_award_moderation_points(self):
         ReviewerScore.award_moderation_points(self.user, self.addon, 1)
         score = ReviewerScore.objects.all()[0]
-        eq_(score.score, amo.REVIEWED_SCORES.get(amo.REVIEWED_ADDON_REVIEW))
-        eq_(score.note_key, amo.REVIEWED_ADDON_REVIEW)
+        assert score.score == amo.REVIEWED_SCORES.get(
+            amo.REVIEWED_ADDON_REVIEW)
```

Comment:
To avoid the weird line-breaking here I'm a fan of:

```python
expected = amo.REVIEWED_SCORES.get(REVIEWED_ADDON_REVIEW)
assert score.score == expected
```